SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql

```diff
@@ -0,0 +1,115 @@
+INSERT INTO help (parentID, hash, topic, content, created, updated) VALUES (-1, md5('acknowledgements'), 'Acknowledgements', 'The Acknowledgements module is designed to serve as a study resource for acknowledging contributors when writing publications. The module stores individual contributor details such as affiliation(s), degree(s), and role(s) within the project, as well as how to cite the contributor’s name for publication.\r\nUsers may search for a specific contributor or group of contributors using Selection Filters for Start and/or End date of their contribution to the study, and whether they are still currently involved in the project, as well as Full Name and Citation Name.', '2017-08-24 00:00:00', NULL);
+
+INSERT INTO `help` (parentID, hash, topic, content, created, updated) VALUES ((SELECT helpID FROM (SELECT * FROM `help`) as temp WHERE hash=md5('survey_accounts')),md5('addSurvey'), 'Add Survey','To create a new survey form for a participant to complete at home, users must enter the DCCID and PSCID of the candidate, then they must select the Visit Label and Instrument name from the corresponding dropdown menus.\r\n\r\n<b>To E-mail a survey directly</b>\r\nIf you wish to send a form within an email to a participant directly, users must enter the e-mail address of the participant  twice, at which point the “Email Survey” button will become active and users may press this button to e-mail the survey.\r\n\r\nOnce the user has clicked “Email Survey”, a pop-up window will appear displaying a generic message that will be sent in the e-mail by default. At this stage, users may edit the message and customize it to fit their needs.','2017-08-24 00:00:00',NULL);
```

Comment:
Is this subselect going to work?

Shouldn't it just be ((SELECT helpID from `help` WHERE hash=md5[...etc]) or am I missing something?